lesson6_php/Form_taikhoan.php

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+        <title>Document</title>
+</head>
+<body>
+    
+        <form action="" method="POST">
+                <h1>ADD PEOPLE</h1>
+                <div class= "mb-3">
+                        <label class="form-label">Họ Tên</label>
+                        <input class="form-control"  type="text" name="name">
+                </div>
+
+                <div class= "mb-3">
+                        <label class="form-label">Username</label>
+                        <input class="form-control" type="text" name="username">
+                </div>
+
+                <div class= "mb-3">
+                        <label class="form-label">Password</label>
+                        <input class="form-control" type="text" name="password">
+                </div>
+
+                <div class= "mb-3">
+        	       <label class="form-label">Email</label>
+        	       <input class="form-control" type="text" name="email">
+                </div>
+
+                <div class= "mb-3">
+        	       <label class="form-label">Số Điện Thoại</label>
+        	       <input class="form-control" type="text" name="phone">
+                </div>
+                
+        	<input class="btn btn-primary" type="submit" name="submit">
+                <input type="text" class="search" name="search">
+                <button value="search">Search</button>
+        </form>
+</body>
+</html>
+
+<?php
+        $servername = "localhost";
+        $username = "root";
+        $password = "";
+        $dbname = "user";
+        $connect = new mysqli($servername, $username, $password, $dbname);
+
+        if ($connect->connect_error) {
+            die("Không kết nối :" . $conn->connect_error);
+            exit();
+        }
+
+        $name = "";
+        $username = "";
+        $password = "";
+        $email = "";
+        $phone = "";
+        $search= "";
+
+        if ($_SERVER["REQUEST_METHOD"] == "POST") {
+                if(isset($_POST["name"])) { $name = $_POST['name']; }
+                if(isset($_POST["username"])) { $username = $_POST['username']; }
+                if(isset($_POST["password"])) { $password = $_POST['password']; }
+                if(isset($_POST["email"])) { $email = $_POST['email']; }
+                if(isset($_POST["phone"])) { $phone = $_POST['phone']; }
+                if(isset($_POST["search"])) { $search = $_POST['search']; }
+
+                $sql = "INSERT INTO account (name, username, password, email, phone)
```

Comment:
```suggestion
                $sql = "INSERT INTO accounts (name, username, password, email, phone)
```
tên bảng nên là danh từ số nhiều nhé